### ISSUE TITLE
Logs say metadata is changed but it doesn't actaully change.

### ISSUE THREAD

**Issue by gageirwin** [JordyAlkema/Youtube-DL-Agent.bundle#21]
When running the agent with some test data almost all of the videos metadata are not being updated even though it says they are in the logs.
Testing on 4 random channels with 5 videos each, only three videos from one of the channel actually has the video metadata changed in plex. The rest of the videos metadata does not change. In the logs they all say processed successfully. It does correctly update the metadata.title and summary but none of the episode data. all videos have a corresponding .info.json and .png thumbnail.

**Comment by Gage-Irwin** [comment]
[com.plexapp.agents.youtube-dl.log](https://github.com/JordyAlkema/Youtube-DL-Agent.bundle/files/9750481/com.plexapp.agents.youtube-dl.log)
![Screenshot 2022-10-10 212025](https://user-images.githubusercontent.com/91108814/194982571-92dc3a29-48dc-4c6d-b8c8-113238cfeba5.png)

**Comment by Gage-Irwin** [comment]
I also just notice that it is also displaying the wrong thumbnails and metadata for the videos that it managed to load metadata to.

**Comment by gageirwin** [comment]
I also noticed that one of the three videos that actually had the metadata applied was incorrect (Underlined in blue) .That video seems to have the metadata of one of the videos that didn't get its metadata updated (Underlined in red).
![Screenshot 2022-10-10 215116](https://user-images.githubusercontent.com/91108814/194986029-51014b88-4dee-426c-a18f-fb1c01e4408a.png)
Here are the logs. 
[com.plexapp.agents.youtube-dl.log](https://github.com/JordyAlkema/Youtube-DL-Agent.bundle/files/9750644/com.plexapp.agents.youtube-dl.log)

**Comment by ShrimpdependenceDay** [comment]
Did you ever figure this out or find an alternative that works? I'm running into the same issue and can't find an agent that updates titles and thumbnails correctly.

**Comment by gageirwin** [comment]
> Did you ever figure this out or find an alternative that works? I'm running into the same issue and can't find an agent that updates titles and thumbnails correctly.

Nope. I even went and read plexes agent documentation and messed with the agent code and it would say it "updated" in the logs but it would never actually update anything in plexes GUI. I spent like 5 days trying and just gave up. If you find an agent that works do let me know.
